core/lib/generators/solidus/install/app_templates/payment_method/braintree.rb

```diff
@@ -0,0 +1,5 @@
+unless Bundler.locked_gems.dependencies['solidus_braintree']
+  bundle_command "add solidus_braintree --git='https://github.com/solidusio/solidus_braintree' --branch='master'"
```

Comment:
The git and branch options are temporary. Once SolidusBraintree 2 is released, we can change this to `"add solidus_braintree ~> 2.0"`